Makefile

```diff
@@ -67,19 +71,29 @@ help:
 
 GOLANGCI_VERSION = 1.46.2
 TRIVY_VERSION= 0.28.1
+PROMTOOL_VERSION = 2.37.0
 TESTS_TIMEOUT="15m"
 
 ##@ General
 
-deps: bin/golangci-lint bin/trivy bin/regcopy bin/jq bin/yq bin/crane ## Install dev dependencies.
+deps: bin/golangci-lint bin/trivy bin/regcopy bin/jq bin/yq bin/crane bin/promtool ## Install dev dependencies.
 
 ##@ Tests
 
-.PHONY: tests-modules tests-matrix tests-openapi
+bin/promtool-${PROMTOOL_VERSION}/promtool:
+	mkdir -p bin/promtool-${PROMTOOL_VERSION}
+	curl -sSfL https://github.com/prometheus/prometheus/releases/download/v${PROMTOOL_VERSION}/prometheus-${PROMTOOL_VERSION}.${GOHOSTOS}-${GOHOSTARCH}.tar.gz | tar zxf - -C bin/promtool-${PROMTOOL_VERSION} --strip=1 prometheus-${PROMTOOL_VERSION}.${GOHOSTOS}-${GOHOSTARCH}/promtool
+
+.PHONY: bin/promtool
+bin/promtool: bin/promtool-${PROMTOOL_VERSION}/promtool
+	rm -f bin/promtool
+	ln -s /deckhouse/bin/promtool-${PROMTOOL_VERSION}/promtool bin/promtool
+
+.PHONY: tests-modules tests-matrix tests-openapi tests-prometheus
```

Comment:
Why we need different versions of promtool ? I think we need only one version, so
```
bin/promtool:
  curl -sSfL https://github.com/prometheus/prometheus/releases/download/v${PROMTOOL_VERSION}/prometheus-${PROMTOOL_VERSION}.${GOHOSTOS}-${GOHOSTARCH}.tar.gz | tar zxf - -C bin --strip=1 prometheus-${PROMTOOL_VERSION}.${GOHOSTOS}-${GOHOSTARCH}/promtool
```

Reply:
I believe we should use the same version of Promtool as the Prometheus version used in Deckhouse to address potential issues with modified rule syntax or new Prometheus features.
This leads to the need of supporting all of the versions of Promtool found across the repository.